example/src/main/java/io/honeycomb/opentelemetry/android/example/MainActivity.kt

```diff
@@ -100,6 +134,8 @@ enum class AnimationSpeed(val sleepTime: Long) {
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun Playground(otel: OpenTelemetryRum?, modifier: Modifier = Modifier) {
+    val apiStatus = remember { mutableStateOf("") }
```

Comment:
Maybe `networkRequestStatus` would be a better name? I'm assuming we'll have more instrumentation in the future.